src/main/resources/templates/static/index.html

```diff
@@ -0,0 +1 @@
+<meta http-equiv="refresh" content="0; URL='/activities'">
```

Comment:
Since the index.html file is pretty empty and they're not using for anything, I thought it would be a good idea to redirect the index page to the activities page. User can still navigate through the site thanks to the navigation bar in the header